### PR TITLE
Fix for reindexing utf-8 in Solr

### PR DESCRIPTION
**Pull request by icarito**
This is what I did to make the Solr reindex.

Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [ ] All tests pass -- `rake test:all`
* [ ] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [ ] pull request are descriptively named
* [ ] if possible, multiple commits squashed if they're smaller changes
* [ ] reviewed/confirmed/tested by another contributor or maintainer
* [ ] `schema.rb.example` has been updated if any database migrations were added

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

**Comment by jywarren** [comment]
Looks good, no?

File: app/models/drupal_node_revision.rb [hunk 1]
```diff
@@ -2,7 +2,10 @@ class DrupalNodeRevision < ActiveRecord::Base
 
   searchable do
     text :title, boost: 5
-    text :body, :teaser
```
Comment (icarito) [inline app/models/drupal_node_revision.rb]: Does this do the right thing?

**Comment by icarito** [comment]
Yes at least it works but I don't know for sure the semantics of this

**Comment by jywarren** [comment]
Well primary search doesn't yet depend on it so I think we should try.

**Comment by jywarren** [comment]
This has been merged. Thanks!

**Pull request closed without merging**